Plugins/Triage/Collect-WindowsArtifacts.ps1

```diff
@@ -0,0 +1,318 @@
+    %SYSTEMROOT%\Appcompat\Programs
+    %SYSTEMROOT%\System32\drivers\etc\hosts
+    %SYSTEMROOT%\System32\winevt\logs
+    %PROGRAMDATA%\Microsoft\Search\Data\Applications\Windows
+    %PROGRAMDATA%\Microsoft\Windows\Start Menu\Programs\Startup
+
+    User Artifacts:
+    %UserProfile%\NTUSER.DAT
+    %UserProfile%\NTUSER.DAT.LOG1
+    %UserProfile%\NTUSER.DAT.LOG2
+    %UserProfile%\AppData\Local\Microsoft\Windows\UsrClass.dat
+    %UserProfile%\AppData\Local\Microsoft\Windows\UsrClass.dat.LOG1
+    %UserProfile%\AppData\Local\Microsoft\Windows\UsrClass.dat.LOG2
+    %UserProfile%\AppData\Roaming\Microsoft\Windows\Recent
+    %UserProfile%\AppData\Local\Google\Chrome\User Data\Default\History
+    %UserProfile%\AppData\Local\Microsoft\Windows\WebCache
+
+.EXAMPLE
+    Stand Alone 
+
+    .\Collect-WindowsArtifacts.ps1 -ComputerName Test-PC
+
+    Power-Response Execution
+
+    Set ComputerName Test-PC
+    Run
+
+.NOTES
+    Author: Matt Weikert
+    Date Created: 2/12/2019
+    Twitter: @5k33tz
+    
+    Last Modified By:
+    Last Modified Date:
+    Twitter:
+  
+#>
+
+param (
+
+    [Parameter(Mandatory=$true,Position=0)]
+    [string[]]$ComputerName
+
+    )
+
+process{
+
+    # Set $Output for where to store recovered artifacts
+    $Output= ("{0}\Artifacts" -f $global:PowerResponse.OutputPath)
+
+    # Create Subdirectory in $global:PowerResponse.OutputPath for storing memory
+    If (-not (Test-Path $Output)) 
+    {
+        New-Item -Type Directory -Path $Output | Out-Null
+    }
+
+    # Verify that velociraptor and 7za executables are located in $global:PowerResponse.Config.Path.Bin
+    $Velociraptor = ("{0}\velociraptor-amd64.exe" -f $global:PowerResponse.Config.Path.Bin)
+    $7za64 = ("{0}\7za_x64.exe" -f $global:PowerResponse.Config.Path.Bin)
+
+    $VelociraptorTestPath = Get-Item -Path $Velociraptor -ErrorAction SilentlyContinue
+    $7z64bitTestPath = Get-Item -Path $7za64 -ErrorAction SilentlyContinue
+
+    if (-not $VelociraptorTestPath) 
+    {
+        Throw "velociraptor-amd64.exe not detected in Bin. Place executable in Bin directory and try again."
+
+    } elseif (-not $7z64bitTestPath) 
+    {
+        Throw "7za_x64.exe not detected in Bin. Place executable in Bin directory and try again."
+    }
+
+    foreach ($Computer in $ComputerName) 
+    {
+        #Handle $ComputerName defined as localhost and not break the Plugin
+        if ($Computer -eq "Localhost") 
+        {
+            $Computer = $ENV:ComputerName
+        }
+
+        #Verify machine is online and ready for data collection
+        if (-not (Test-Connection -ComputerName $Computer -Count 1 -Quiet)) 
+        {
+            Write-Error ("{0} appears to be offline. Cannot collect artifacts." -f $Computer)
+            Continue
+        }
+
+        $7zaInstallexe = $7za64
+
+        #Establish PS Session criteria
+        $Session = New-PSSession -ComputerName "$Computer" -SessionOption (New-PSSessionOption -NoMachineProfile)
```

Comment:
Could use this `$Session` for the EXE copies and for the `Invoke-Command` calls later on for more secure credential handling.